docs/development/virtual_environments.rst

```diff
@@ -171,6 +170,14 @@ libvirt boxes:
    valid provider. In this case, remove Vagrant with ``sudo apt-get remove
    vagrant`` and reinstall it.
 
+Log out, then log in again. Verify that libvirt is installed and KVM is available:
+
+.. code:: sh
+  
+   libvirtd --version
+   kvm-ok
```

Comment:
Is this (`kvm-ok`) a command? Because when I followed the docs originally, this wasn't installed.

Reply:
Ah wait that's Ubuntu. You removed it on the Debian ones. Nvm.

Reply:
Yep!